lib/modules/userInfo.js

```diff
@@ -97,33 +97,18 @@ addModule('userInfo', function (module, moduleID) { $.extend(true, module, {
 			.begin();
 	},
 	showUserInfo: function(def, obj, context) {
-		def.notify('Loading...', '<span class="RESThrobber />');
-		var isFriend = obj.classList.contains('friend'),
-			test = obj.href.match(modules['userTagger'].usernameRE),
-			thisUserName;
+		var test = obj.href.match(modules['userTagger'].usernameRE),
+			thisUserName,
+			throbber = '<span class="RESThrobber"></span>';
 
 		if (test) {
 			thisUserName = test[1];
 		}
 		module.lastOpenedAuthorLink = obj;
-
-		var header = $('<div />');
-		header.append('<a href="/user/' + escapeHTML(thisUserName) + '">/u/' + escapeHTML(thisUserName) + '</a> (<a href="/user/' + escapeHTML(thisUserName) + '/submitted/">Links</a>) (<a href="/user/' + escapeHTML(thisUserName) + '/comments/">Comments</a>)');
-		RESUtils.getUserInfo(function(userInfo) {
-			var friendButton, myID;
-			myID = 't2_' + userInfo.data.id;
-			if (isFriend) {
-				friendButton = '<span class="fancy-toggle-button toggle" style="display: inline-block; margin-left: 12px;"><a class="option active remove" href="#" tabindex="100" onclick="return toggle(this, unfriend(\'' + obj.textContent + '\', \'' + myID + '\', \'friend\'), friend(\'' + obj.textContent + '\', \'' + myID + '\', \'friend\'))">- friends</a><a class="option add" href="#">+ friends</a></span>';
-			} else {
-				friendButton = '<span class="fancy-toggle-button toggle" style="display: inline-block; margin-left: 12px;"><a class="option active add" href="#" tabindex="100" onclick="return toggle(this, friend(\'' + obj.textContent + '\', \'' + myID + '\', \'friend\'), unfriend(\'' + obj.textContent + '\', \'' + myID + '\', \'friend\'))">+ friends</a><a class="option remove" href="#">- friends</a></span>';
-			}
-			header.append(friendButton);
-			def.notify(header, null);
-		});
-
-		def.notify(header, null);
+		def.notify(thisUserName, throbber);
 
 		RESUtils.cache.fetch({
+			key: thisUserName.toLowerCase(),
```

Comment:
In this case, this is unnecessary since the endpoint URL will be different for each user (if no key is specified, the key is generated from the endpoint - e.g. `RESUtils.cache.-user-erikdesjardins-about-json`).